src/components/MultiHopTrade/components/FiatRamps/hooks/useGetRampQuotes.tsx

```diff
@@ -40,10 +38,8 @@ export const useGetRampQuotes = ({
   const debouncedAmount = useDebounce(amount, 1000)
 
   const queryKey = useMemo(() => {
-    dispatch(tradeRampInput.actions.setSelectedFiatRampQuote(null))
-
-    return ['rampQuote', fiatCurrency, assetId, debouncedAmount, direction, onramperCurrencies]
```

Comment:
q: is this expected? Looks like we may still want `fiatCurrency` and `assetId` as deps?

Reply:
I think it's expected, if we add those query keys, the query will reload